dspace-spring-rest/src/test/java/org/dspace/app/rest/RelationshipTypeRestControllerIT.java

```diff
@@ -0,0 +1,212 @@
+                                   .build();
+
+        Item project1 = ItemBuilder.createItem(context, col3)
+                                   .withTitle("Project1")
+                                   .withAuthor("Testy, TEst")
+                                   .withIssueDate("2015-01-01")
+                                   .withRelationshipType("Project")
+                                   .build();
+
+        Item publication = ItemBuilder.createItem(context, col3)
+                                      .withTitle("Publication1")
+                                      .withAuthor("Testy, TEst")
+                                      .withIssueDate("2015-01-01")
+                                      .withRelationshipType("Publication")
+                                      .build();
+
+        Item publication2 = ItemBuilder.createItem(context, col3)
+                                       .withTitle("Publication2")
+                                       .withIssueDate("2015-01-01")
+                                       .withRelationshipType("Publication")
+                                       .build();
+
+        RelationshipType isOrgUnitOfPersonRelationshipType = relationshipTypeService
+            .findbyTypesAndLabels(context, entityTypeService.findByEntityType(context, "Person"),
+                                  entityTypeService.findByEntityType(context, "OrgUnit"),
+                                  "isOrgUnitOfPerson", "isPersonOfOrgUnit");
+        RelationshipType isOrgUnitOfProjectRelationshipType = relationshipTypeService
+            .findbyTypesAndLabels(context, entityTypeService.findByEntityType(context, "Project"),
+                                  entityTypeService.findByEntityType(context, "OrgUnit"),
+                                  "isOrgUnitOfProject", "isProjectOfOrgUnit");
+        RelationshipType isAuthorOfPublicationRelationshipType = relationshipTypeService
+            .findbyTypesAndLabels(context, entityTypeService.findByEntityType(context, "Publication"),
+                                  entityTypeService.findByEntityType(context, "Person"),
+                                  "isAuthorOfPublication", "isPublicationOfAuthor");
+
+        Relationship relationship1 = RelationshipBuilder
+            .createRelationshipBuilder(context, publication, author1, isAuthorOfPublicationRelationshipType).build();
+        Relationship relationship2 = RelationshipBuilder
+            .createRelationshipBuilder(context, publication, author2, isAuthorOfPublicationRelationshipType).build();
+        Relationship relationship3 = RelationshipBuilder
+            .createRelationshipBuilder(context, publication2, author2, isAuthorOfPublicationRelationshipType).build();
+        Relationship relationship4 = RelationshipBuilder
+            .createRelationshipBuilder(context, publication2, author3, isAuthorOfPublicationRelationshipType).build();
+
```

Comment:
please restore the authz system

Reply:
Included in https://github.com/DSpace/DSpace/pull/2418